french/maps/conv/evelyn_convs.rpy

```diff
@@ -4,131 +4,131 @@
 translate french evelyn_outback_fields_6a5aeb55:
 
     # d surprise "You alright there, love?"
-    d surprise ""
+    d surprise "Tu vas bien, mon amour ?"
 
 # game/maps/conv/evelyn_convs.rpy:18
 translate french evelyn_outback_fields_443e059a:
 
     # e nulla "Yeah, yeah, just...{w=0.5} It's taking me a while to get used to the weather and the...{w=0.5} Vastness of it."
-    e nulla ""
+    e nulla "Ouais, ouais, juste...{w=0.5} Il me faut du temps pour m'habituer à la météo et à...{w=0.5} Son immensité."
 
 # game/maps/conv/evelyn_convs.rpy:20
 translate french evelyn_outback_fields_8db2f802:
 
     # e nulla "It's a completely alien environment for a poor mountain bunny like me...{w=0.5} Even while doing Orphan work for Ozpin, I never left the Valesian half of the continent."
-    e nulla ""
+    e nulla "C'est un environnement complètement étranger pour une pauvre lapine des montagnes comme moi...{w=0.5} Même en faisant du travail d'Orphelin pour Ozpin, je n'ai jamais quitté la moitié Valésienne du continent."
 
 # game/maps/conv/evelyn_convs.rpy:21
 translate french evelyn_outback_fields_d2e573ae:
 
     # d amused "Thank god for creative applications of Aura...{w=0.5} And Polendina technology."
-    d amused ""
+    d amused "Dieu merci pour les applications créatives de l'Aura...{w=0.5} et la technologie Polendina."
 
 # game/maps/conv/evelyn_convs.rpy:23
 translate french evelyn_outback_fields_2ce66643:
 
     # e nulla "Sure, rub your fancy suit in my face...{w=0.5} I swear I'll steal it the next time you get naked."
-    e nulla ""
+    e nulla "C'est sur, frotte-moi ta combinaison de luxe au visage...{w=0.5} Je jure de la voler la prochaine fois que tu seras nu."
```

Comment:
to rub X in my face -> se la ramener auprès de moi/ remuer le couteau dans la plaie (en anglais ça fait les deux)
"Bien sûr, ramène-la, toi, avec ta combinaison de luxe..."

la seconde phrase est pas bonne parce qu'en anglais on omet "that" I swear THAT I'll steal it [...]
Je jure que je vais te la voler [...]